src/Func.cpp

```diff
@@ -1732,6 +1732,14 @@ Stage &Stage::prefetch(const Internal::Parameter &param, VarOrRVar var, Expr off
     return *this;
 }
 
+/** Attempt to get the source file and line where this stage was
```

Comment:
I like this wording a little more than the one in Definition.h which mentions DWARF explicitly. (Making introspection work on Windows would invalidate the DWARF part right?)